README.md

```diff
@@ -18,6 +18,47 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #0xx - "use strict" and get lazy
+
+> 2016-xx-xx by [@nainslie](https://twitter.com/nat5an)
+
+By default, JavaScript allows the programmer to be pretty careless, for example, by not requiring us to declare our variables with "var" when we first introduce them.  While this may seem like a convenience to the unseasoned developer, it's also the source of many errors when a variable name is misspelled or accidentally referred to out of its scope.
+
+Programmers like to make the computer do the boring stuff for us, and automatically check our work for mistakes. That's what the JavaScript "use strict" directive allows us to do, by turning our mistakes into JavaScript errors. 
+
+We add this directive either by adding it at the top of a js file: 
+
+```javascript
+// Whole-script strict mode syntax
+"use strict";
+var v = "Hi!  I'm a strict mode script!";
+```
+
+or inside a function:
+
+```javascript
+function strict()
+{
+  // Function-level strict mode syntax
+  'use strict';
+  function nested() { return "And so am I!"; }
+  return "Hi!  I'm a strict mode function!  " + nested();
+}
+function notStrict() { return "I'm not strict."; }
+```
+
+By including this directive in a JavaScript file or function, we will direct the JavaScript engine to execute in strict mode which disables a bunch of behaviors that are usually undesirable in larger JavaScript projects.  Among other things, strict mode changes the following behaviors:
+* Variables can only be introduced when they are preceded with "var"
+* Attempting to write to readonly properties generates a noisy error
+* You have to call constructors with the "new" keyword
+* "this" is not implicitly bound to the global object 
+* Very limited use of eval() allowed
+
+Strict mode is great for new projects, but can be challenging to introduce into older projects that don't already use it in most places.  It also can be problematic if your build chain concatenates all your js files into one big file, as this may cause all files to execute in strict mode.
+
```

Comment:
I never read about remove "use strict" in production but for example in babel is unnecessary